app/controllers/hubstats/teams_controller.rb

```diff
@@ -33,7 +33,7 @@ def show
       @team = Hubstats::Team.where(id: params[:id]).first
       @pull_requests = Hubstats::PullRequest.belonging_to_team(@team.id).merged_in_date_range(@start_date, @end_date).order("updated_at DESC").limit(20)
       @pull_count = Hubstats::PullRequest.belonging_to_team(@team.id).merged_in_date_range(@start_date, @end_date).count(:all)
-      @users = @team.users.where("login NOT IN (?)", Hubstats.config.github_config["ignore_users_list"]).order("login ASC")
+      @users = @team.users.where("login NOT IN (?)", Hubstats.config.github_config["ignore_users_list"]).order("login ASC").uniq
```

Comment:
Can we use distinct so that the uniqueness is processed in SQL and not in Ruby.  This would avoid loading lots of ActiveRecord objects in memory and less transfer from the DB to the app.

http://apidock.com/rails/ActiveRecord/QueryMethods/distinct